tests-manual/bigDash.R

```diff
@@ -0,0 +1,313 @@
+if(interactive()) {

```

Comment:
This app looks like this, which doesn't seem right. Now that I look at the code, shouldn't everything in the body be contained in `tabItem`s?

![image](https://cloud.githubusercontent.com/assets/86978/12572523/335d8892-c3b3-11e5-9b3e-d825ef0b5e8c.png)